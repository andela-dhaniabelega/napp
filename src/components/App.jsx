import React from 'react';
import BodyStyle from 'body-style';
import '../../public/sass/styles.scss'; 

/**
 * Stateless component
 * Sets Body style
 *
 * @return {void}
 */
const App = () => (
  <div>
    <BodyStyle className="wrapper" />
  </div>
);

export default App;