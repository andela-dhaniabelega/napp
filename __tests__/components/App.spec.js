import React from 'react';
import renderer from 'react-test-renderer'
import App from '../../src/components/App';


describe('Contact Us Page', () => {
		it('renders a snapshot', () => {
          const tree = renderer.create(<App />).toJSON();
          expect(tree).toMatchSnapshot();
		});
});