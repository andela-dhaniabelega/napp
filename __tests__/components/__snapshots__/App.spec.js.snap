// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Us Page renders a snapshot 1`] = `<div />`;
